src/lib.rs

```diff
@@ -21,29 +21,55 @@
 //! Zerocopy makes zero-cost memory manipulation effortless. We write `unsafe`
 //! so you don't have to.
 //!
+//! [customer-request-issue]: https://github.com/google/zerocopy/issues/new/choose
+//!
 //! # Overview
 //!
-//! Zerocopy provides four core marker traits, each of which can be derived
-//! (e.g., `#[derive(FromZeros)]`):
-//! - [`TryFromBytes`] indicates that a type may safely be converted from certain
-//!   byte sequences (conditional on runtime checks)
+//! ###### Conversion Traits
```

Comment:
Why so many pound signs?

Reply:
H6 is the lowest level of heading tag, used here because I want paragraph-level headings, not section-level headings.